TeamCode/src/main/java/org/firstinspires/ftc/teamcode/roadrunner/drive/DriveConstants.java

```diff
@@ -64,18 +64,17 @@ public class DriveConstants {
      * small and gradually increase them later after everything is working. All distance units are
      * inches.
      */
-    public static double MAX_VEL = 30;
-    public static double MAX_ACCEL = 30;
-    public static double MAX_ANG_VEL = Math.toRadians(60);
-    public static double MAX_ANG_ACCEL = Math.toRadians(60);
-
+    public static double MAX_VEL = 49.95; // 90% of 52.17, the actual measured max velocity
```

Comment:
We will likely need to re-tune this value pretty soon because adding heavy parts changes it.